medmodels/matching/algorithms/propensity_score.py

```diff
@@ -1,24 +1,27 @@
 from __future__ import annotations
 
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Any, Dict, Literal, Optional, Tuple, Union
 
 import numpy as np
-import pandas as pd
+import polars as pl
 from numpy.typing import NDArray
 from sklearn.ensemble import RandomForestClassifier
 from sklearn.linear_model import LogisticRegression
 from sklearn.tree import DecisionTreeClassifier
+from typing_extensions import TypeAlias
```

Comment:
See https://github.com/limebit/medmodels/blob/main/medmodels/medrecord/types.py#L8-L14 on how to import this correctly